src/Compilers/Core/CodeAnalysisTest/Resources.Designer.cs

```diff
@@ -59,45 +59,5 @@ internal class Resources {
                 resourceCulture = value;
             }
         }
-        
-        /// <summary>
-        ///   Looks up a localized resource of type System.Byte[].
-        /// </summary>
-        internal static byte[] defaultWin32Manifest {
```

Comment:
I don't see the corresponding change to `resx` file (which generates this `cs` file).